service/src/main/resources/db/changesets/20240412-testdata.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: add data for unit tests
+      author: mma
+      context: test
+      changes:
+        - insert:
+            columns:
+              - column:
+                  name: job_id
+                  value: deadbeef-dead-beef-deaf-beefdeadbeef
+              - column:
+                  name: user_id
+                  value: testUser
+              - column:
+                  name: pipeline_id
+                  valueComputed: (SELECT id FROM pipelines WHERE name='imputation_beagle')
```

Comment:
oh interesting, might be worth doing something similar in the other test data file to make sure its getting the correct pipeline_id value.  I'm honestly surprised the tests still work after deleting the minimac pipeline from that table

Reply:
actually i think we can delete the other test data file entirely